oaknational/Oak-Web-Application
# [CUR-260] Add MV to curriculum overview

## Description

- Updates front end to include curriculum intent statement from DB
- Only uses subject/phase
- Primary music and secondary geography will result in a 404 as these are not available


## How to test

1. Go to https://deploy-preview-1913--oak-web-application.netlify.thenational.academy/teachers/curriculum
2. Select any subject phase combination
3. You should see the correct curriculum intent statement

## Screenshots

How it used to look (delete if n/a):
![Screenshot 2023-09-19 at 16 54 25](https://github.com/oaknational/Oak-Web-Application/assets/32605982/8e96f30e-2293-4111-a6ee-9da4ecef2e11)

How it should now look:
![Screenshot 2023-09-19 at 16 53 45](https://github.com/oaknational/Oak-Web-Application/assets/32605982/0182227f-4e51-4357-9111-2ef54100156d)


## Checklist
